### DISCUSSION TITLE
Svelte and reactivity of operationStore

### DISCUSSION THREAD

**Discussion by thojanssens** [urql-graphql/urql#2312] [Q&A]
When a prop passed from the parent changes, and is used as a parameter of an urql query, the query does not seem to be reexecuted.

I see many examples such as:

```javascript
export let id;
const store = operationStore(query, { id })
query(store)
```

But why there doesn't seem to be any example available, where we want to reexecute the query simply because the prop (variable id in example above) changed? It seems a very common case.

I must be missing something big or the way I design my components is wrong?

**Comment by frederikhors** [comment]
https://formidable.com/open-source/urql/docs/basics/svelte/

**Comment by frederikhors** [comment]
https://github.com/FormidableLabs/urql/issues/2080

**Comment by frederikhors** [comment]
https://github.com/FormidableLabs/urql/issues/2195
